app/src/main/java/org/sil/storyproducer/model/Phase.kt

```diff
@@ -3,146 +3,169 @@ package org.sil.storyproducer.model
 import org.sil.storyproducer.R
 import org.sil.storyproducer.controller.MainActivity
 import org.sil.storyproducer.controller.RegistrationActivity
-import org.sil.storyproducer.controller.export.CreateActivity
+import org.sil.storyproducer.controller.export.FinalizeActivity
 import org.sil.storyproducer.controller.export.ShareActivity
 import org.sil.storyproducer.controller.learn.LearnActivity
 import org.sil.storyproducer.controller.pager.PagerBaseActivity
 import org.sil.storyproducer.controller.remote.WholeStoryBackTranslationActivity
 
-
 enum class PhaseType {
-    WORKSPACE, REGISTRATION, STORY_LIST, LEARN, DRAFT, COMMUNITY_CHECK, CONSULTANT_CHECK, DRAMATIZATION, CREATE, SHARE, BACKT, WHOLE_STORY, REMOTE_CHECK
+    WORKSPACE,
+    REGISTRATION,
+    STORY_LIST,
+    LEARN,
+    TRANSLATE_REVISE,
+    COMMUNITY_WORK,
+    ACCURACY_CHECK,
+    VOICE_STUDIO,
+    FINALIZE,
+    SHARE,
+    BACK_TRANSLATION,

```

Comment:
Just a matter of style... My preference would be to just keep "BACKT" as the enum values, since it's brief, it's not a released feature yet, another core phase already has the word "TRANSLATE", and "BACK_TRANSLATION" becomes the longest enum of the set. 
Or how about "BACK_T"? That clear. What do you think?

Reply:
I agree with you on this. Let's avoid using the word 'translation' again. I'll make the fix